src/data/set/finite.lean

```diff
@@ -72,6 +72,15 @@ theorem exists_finite_iff_finset {p : set α → Prop} :
 ⟨λ ⟨s, hs, hps⟩, ⟨hs.to_finset, hs.coe_to_finset.symm ▸ hps⟩,
   λ ⟨s, hs⟩, ⟨↑s, finite_mem_finset s, hs⟩⟩
 
+lemma finite.fin_param {s : set α} (h : finite s) :
+  ∃ (n : ℕ) (f : fin n → α), injective f ∧ range f = s :=
```

Comment:
A better statement might be
```suggestion
  ∃ (n : ℕ) (f : fin n ↪ α), range f = s :=
```

Reply:
I thought about it but I preferred the unbundled version. We could probably have both.